translations/fr/calendar.yml

```diff
@@ -1,6 +1,6 @@
 January: janvier
 February: février
-March: Mars
+March: mars
```

Comment:
Looking [here](https://www.fluentu.com/blog/french/french-months-of-the-year/) all months should begin with lower case so think this looks good